modules/dnn/test/test_onnx_conformance_layer_filter__openvino.inl.hpp

```diff
@@ -1588,6 +1588,10 @@ CASE(test_scatter_elements_with_duplicate_indices)
     // no filter
 CASE(test_scatter_elements_with_negative_indices)
     // no filter
+CASE(test_scatter_elements_with_reduction_max)
```

Comment:
Most probably the tests for scatter should be disabled for CUDA, Vulcan, OpenCL back-ends too. Also it's not clear, if it works works with Tim-VX.

Reply:
Added into the deny lists of cuda, halide, vulkan and opencl. As for TIM-VX, it does not have a deny list yet and this could be done in a separate pull request.